### PR TITLE
T472

### PR DESCRIPTION
**Pull request by secretrobotron**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @secretrobotron received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/df58d2d9436d7c4/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/df58d2d9436d7c4/output.txt

Total script time: 0.49 mins

Server is running on http://pr331.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @brettgaylor received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/b3257bff51529de/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/b3257bff51529de/output.txt

Total script time: 0.50 mins

Server is running on http://pr331.butyr.org/templates/test.html

File: src/timeline/media.js [hunk 1]
```diff
@@ -70,6 +70,14 @@ define( [
...
     _mediaStatusContainer.className = "media-status-container";
 
+    function snapToCurrentTime(){
+      _tracksContainer.snapTo( _media.currentTime );
+      _hScrollBar.update();
+    }
+
+    _media.listen( "mediaplaying", snapToCurrentTime );
+    _media.listen( "mediapause", snapToCurrentTime );
+
```
Comment (cadecairos) [inline src/timeline/media.js]: lets remove this extra white space here

**Pull request merged**